action.yml

```diff
@@ -159,18 +162,21 @@ runs:
           echo "Set NIX_INSTALLER_NIX_PACKAGE_URL=$NIX_INSTALLER_NIX_PACKAGE_URL"
         fi
 
+        nix_extra_conf=""
+        newline='
+        '
         if [ -n "${{ inputs.extra-conf }}" ]; then
-          if [ -n "${{ inputs.github-token }}" ]; then
-            export NIX_INSTALLER_EXTRA_CONF="${{ inputs.extra-conf }}access-tokens = github.com=${{ inputs.github-token }}"
-          else
-            export NIX_INSTALLER_EXTRA_CONF="${{ inputs.extra-conf }}"
-          fi
+          nix_extra_conf="${{ inputs.extra-conf }}"
```

Comment:
Neat method. :) Do you think we can make `NEWLINE` and `NIX_EXTRA_CONF` all caps just for my own sake? :sweat_smile:

Reply:
Sure, fixed!